STARTERKIT/.sass-lint.yml

```diff
@@ -40,7 +40,7 @@ rules:
   # no-css-comments: 2
   no-debug: 2
   # no-duplicate-properties: 2
-  no-empty-rulesets: 2
+  no-empty-rulesets: 1
```

Comment:
Why setting it to a warning instead of an error? No need to keep empty rules.

https://github.com/sasstools/sass-lint/blob/master/docs/rules/no-empty-rulesets.md